applications/bouncer-live.yaml

```diff
@@ -10,7 +10,11 @@ spec:
   source:
     repoURL: https://github.com/mindtastic/deployments
     targetRevision: HEAD
-    path: bouncer/live
+    path: bouncer
+    helm:
+      releaseName: bouncer
+      valueFiles:
+      - values-live.yaml    
   destination:
-    server: https://kubernetes.default.svc
+    server: https://cluster.net.dev.mindtastic.lol
```

Comment:
Should we keep the `https://kubernetes.default.svc` since it's live yaml file? @ldb